_docs/2.0.0/usage/run.md

```diff
@@ -133,8 +138,7 @@ The list of options for configuring Spark deployment mode in Yarn and resource s
 | --conf-spark-memory-fraction **value**                               |             | **Advanced**. Fraction of (heap space - 300MB) used for execution and storage (default=`0.6`). Sets `spark.memory.fraction` Spark configuration parameter. See the detailed description [here](http://spark.apache.org/docs/latest/configuration.html#memory-management) |
 
 
-For more information on these options see the official documentation on running Spark on Yarn: 
-[https://spark.apache.org/docs/latest/running-on-yarn.html](https://spark.apache.org/docs/latest/running-on-yarn.html)
+For more information on these options see the official documentation on [running Spark on Yarn] [spark-running-yarn]
```

Comment:
```suggestion
For more information on these options see the official documentation on [running Spark on Yarn][spark-running-yarn]
```

Does it work with the space?

Reply:
Surprisingly it does. I will still consider fixing it. Good catch. 🙇